benchmarks/decoders/benchmark_decoders_library.py

```diff
@@ -111,6 +121,18 @@ def get_consecutive_frames_from_video(self, video_file, numFramesToDecode):
             frames.append(frame["data"].permute(1, 2, 0))
         return frames
 
+    def decode_and_transform(self, video_file, pts_list, height, width, device):
+        self.torchvision.set_video_backend(self._backend)
+        reader = self.torchvision.io.VideoReader(video_file, "video")
+        frames = []
+        for pts in pts_list:
+            reader.seek(pts)
+            frame = next(reader)
+            frames.append(frame["data"].permute(1, 2, 0))
+        frames = [frame.to(device) for frame in frames]
+        frames = self.transforms_v2.functional.resize(frames, (height, width))
```

Comment:
Naive question that applies to all implementations that use the transformation: how do we ensure it's done on the GPU?

Reply:
Correct